src/index.js

```diff
@@ -533,7 +537,7 @@ export default class InputNumber extends React.Component {
   // '1.0' '1.00'  => may be a inputing number
   toNumberWhenUserInput(num) {
```

Comment:
Do we still need this method?

Reply:
No, we do not. I have deleted that method and I converting straight to the number.